routes/Login.py

```diff
@@ -8,5 +8,9 @@ def __init__(self, path, methods) -> None:
     def getResponse(self, request: Request):
         if request.req_type == "POST":
             return Route.buildResponse()
+        elif request.req_type == "GET":
+            f = open("./public/login.html", "rb")
+            file = f.read()
+            file_length = str(len(file))
```

Comment:
don't need to set content length explicitly, responseBuilder does it under the hood